File: BackEnd/Models/Portao.cs
using System.ComponentModel.DataAnnotations;

public class Portao
{
    [Key]
    public int Id { get; set; }
    public string? Codigo { get; set; }
    public Aeroporto? Aeroporto { get; set; }
    public int AeroportoId { get; set; }
}

File: FrontEnd/AppPassagem/src/app/models/Portao.ts
import { Aeroporto } from "./Aeroporto";

export class Portao {
    id: number = 0;
    codigo : string = "";
    aeroporto : Aeroporto;
    aeroportoId : number = 0;

    constructor(aeroporto: Aeroporto){
        this.aeroporto = aeroporto;
    }
}